src/Model/VideoManager.php

```diff
@@ -2,25 +2,24 @@
 
 namespace Mi\VideoManagerPro\SDK\Model;
 
+use JMS\Serializer\Annotation\Type;
+
 /**
  * @author Steve Reichenbach <steve.reichenbach@movingimage.com>
  *
  * @codeCoverageIgnore
  */
 class VideoManager
 {
+    /**
+     * @Type("string")
+     */
     private $id;
-    private $name;
 
     /**
-     * @param $name
-     * @param $id
+     * @Type("string")
      */
-    public function __construct($name, $id)
```

Comment:
I do not know why this constructor was there but maybe analytics need it - have you checked it!?